lib/web/css/source/lib/_navigation.less

```diff
@@ -99,7 +99,7 @@
 
             &.active {
                 .all-category {
-                    .ui-state-focus {
+                    .ui-state-active {
```

Comment:
We can change the class name in backward backward-compatible way like the below:

.ui-state-active, .ui-state-focus